test/lib.bash

```diff
@@ -269,9 +269,13 @@ function gather_knative_state {
   logger.info 'Gather knative state'
   local gather_dir="${ARTIFACT_DIR:-/tmp}/gather-knative"
   mkdir -p "$gather_dir"
+  IMAGE_OPTION="--image=quay.io/openshift-knative/must-gather"
+  if [[ $FULL_MESH == true ]]; then
+    IMAGE_OPTION="${IMAGE_OPTION} --image=registry.redhat.io/openshift-service-mesh/istio-must-gather-rhel7"
+  fi
```

Comment:
Will this run both must gathers?

Reply:
Yes, help option says that and I verified it on my local.

```
$ oc adm must-gather -h

  # gather information using multiple plug-in images
  oc adm must-gather --image=quay.io/kubevirt/must-gather
--image=quay.io/openshift/origin-must-gather
```